cognite/client/_api/data_modeling/instances.py

```diff
@@ -789,7 +787,9 @@ async def subscribe(
             Subscribe to a given query and process the results:
 
                 >>> from cognite.client import AsyncCogniteClient
-                >>> from cognite.client.data_classes.data_modeling.query import Query, NodeResultSetExpression, Select, SourceSelector
+                >>> from cognite.client.data_classes.data_modeling.query import (
+                >>>     SyncQuery, NodeResultSetExpressionSync, SelectSync, SourceSelector
```

Comment:
Should be `QuerySync`? I would have expected this to be picked up by the doctest tests 🤔

Reply:
Though I think I like `SyncQuery` better.

Reply:
It's a trade-off with consistent naming, I guess. `SyncQuery` is more descriptive in this case, but i don't like `SyncNodeResultSetExpression`.